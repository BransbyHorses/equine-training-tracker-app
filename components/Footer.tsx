import React from "react";
import { Box } from "@mui/material";

const Footer = () => {
	return (
		<Box sx={{ height: "75px" }}>
			<footer></footer>
		</Box>
	);
};

export default Footer;
