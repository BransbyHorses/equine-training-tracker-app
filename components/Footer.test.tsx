import renderer from "react-test-renderer";
import "@testing-library/jest-dom";
import Footer from "./Footer";

describe("Footer", () => {
	it("renders", () => {
        const tree = renderer.create(<Footer />);
		expect(tree).toMatchSnapshot();
	});
});
