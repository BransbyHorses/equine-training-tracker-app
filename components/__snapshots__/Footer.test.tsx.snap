// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<div
  className="MuiBox-root css-1x6rvtb"
>
  <footer />
</div>
`;
